integration-tests/README.md

```diff
@@ -45,10 +45,13 @@ The following environment variables are parsed by the tests:
 #### Convenient one-liner:
 
 ```sh
+[HEADLESS=false] \
+[ENVIRONMENT=local|dev|alpha|perf|staging] \
```

Comment:
Adding these in this format makes this command no longer paste-able. Perhaps we should make sure the options are documented sufficiently above and let this be a simple example.

Reply:
Breaking copy/paste did cross my mind. I'm glad you weighed in. The full documentation of all of the options above is a bit heavy for someone looking for a convenient one-liner but could still benefit from a reminder of these two options in particular. I'll pull them out of the `sh` block and just mention them in this section.